test/api/generateXAddress/index.ts

```diff
@@ -1,6 +1,7 @@
 import assert from 'assert-diff'
 import responses from '../../fixtures/responses'
 import {TestSuite} from '../../utils'
+import ECDSA from '../../../src/common/types/enums/ecdsa'
 import {GenerateAddressOptions} from '../../../src/offline/generate-address'
```

Comment:
Do we want to change `GenerateAddressOptions` to `WalletOptions` or `GenerateWalletOptions`?

Reply:
I'm not sure. I personally think it should stay as `GenerateAddressOptions` for now since it's being used in the context of `generateAddressAPI()`.